ait/core/server/server.py

```diff
@@ -89,6 +95,73 @@ def _load_streams(self):
         if not self.outbound_streams:
             log.warn(err_msgs['outbound'])
 
+    def _create_api_telem_stream(self):
+        """"""
+        stream_map = {'__valid_api_streams': []}
+        COMPATIBLE_HANDLERS = ['PacketHandler', 'CCSDSPacketHandler']
```

Comment:
@MJJoyce Is this a TODO-make more configurable? Or is this expected to the the standard?

Reply:
I think I took out the TODO that I had commented in here. Guess I should have left it in =D

So this list is only the list of handlers that we're using when we try to set a default if nothing is configured. Eventually I'd like to see it (potentially) moved somewhere else that makes more sense or figure out a better way for determining output format compatibility in handlers. Or something else entirely that we can think of that makes more sense. Being able to check handler output compatibility would be nice so we could verify that the configuration is good even when the user is specifying it.

I'm not really what the best approach is though for the time being. I guess we could pull it from constant somewhere so it's easier for users to modify? Of course, if you're going through that much trouble you'd be better off just specifying what streams to use in the config.